### PR TITLE
Adding a mention to Zone Hold

### PR DESCRIPTION
**Pull request by ngayerie**
Adding a mention to Zone Hold as it may prevent a custom subdomain to activate

### CODE REVIEW

File: content/pages/platform/custom-domains.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ To use a custom apex domain (for example, `example.com`) with your Pages project
 
 If you are deploying to a subdomain, it is not necessary for your site to be a Cloudflare zone. You will need to [add a custom CNAME record](#add-a-custom-cname-record) to point the domain to your Cloudflare Pages site. To deploy your Pages project to a custom apex domain, that custom domain must be a zone on the Cloudflare account you have created your Pages project on.
 
+{{<Aside type="note">}}
+
+If the zone is on the Enterprise plan, please make sure that the you [release the Zone Hold feature](/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) before adding the custom domain, as the Zone Hold feature would prevent the custom subdomain to activate.
```
Comment (deadlypants1973) [inline content/pages/platform/custom-domains.md]:
```suggestion
If the zone is on the Enterprise plan, make sure that the you [release the zone hold](/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) before adding the custom domain. A zone hold would prevent the custom subdomain to activate.
```
Adjusting wording and caps to match fundamentals page.

**Review by deadlypants1973 — APPROVED** [review]
PCX review

**Comment by WalshyDev** [comment]
Thank you Nic!

**Review by WalshyDev — APPROVED** [review]

File: content/pages/platform/custom-domains.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ To use a custom apex domain (for example, `example.com`) with your Pages project
 
 If you are deploying to a subdomain, it is not necessary for your site to be a Cloudflare zone. You will need to [add a custom CNAME record](#add-a-custom-cname-record) to point the domain to your Cloudflare Pages site. To deploy your Pages project to a custom apex domain, that custom domain must be a zone on the Cloudflare account you have created your Pages project on.
 
+{{<Aside type="note">}}
+
+If the zone is on the Enterprise plan, make sure that the you [release the zone hold](/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) before adding the custom domain. A zone hold would prevent the custom subdomain to activate.
```
Comment (Skye-31) [inline content/pages/platform/custom-domains.md]: Minor grammar note - the current wording doesn't make the most sense 

```suggestion
If the zone is on the Enterprise plan, make sure that the you [release the zone hold](/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) before adding the custom domain. A zone hold would prevent the custom subdomain from activating.
```

**Pull request merged**